pkg/security/probe/process_resolver.go

```diff
@@ -242,14 +244,39 @@ func (p *ProcessResolver) NewProcessCacheEntry() *model.ProcessCacheEntry {
 
 // SendStats sends process resolver metrics
 func (p *ProcessResolver) SendStats() error {
-	if err := p.client.Gauge(metrics.MetricProcessResolverCacheSize, p.GetCacheSize(), []string{}, 1.0); err != nil {
+	var err error
+	var count int64
+
+	if err = p.client.Gauge(metrics.MetricProcessResolverCacheSize, p.GetCacheSize(), []string{}, 1.0); err != nil {
 		return errors.Wrap(err, "failed to send process_resolver cache_size metric")
 	}
 
-	if err := p.client.Gauge(metrics.MetricProcessResolverReferenceCount, p.GetEntryCacheSize(), []string{}, 1.0); err != nil {
+	if err = p.client.Gauge(metrics.MetricProcessResolverReferenceCount, p.GetEntryCacheSize(), []string{}, 1.0); err != nil {
 		return errors.Wrap(err, "failed to send process_resolver reference_count metric")
 	}
 
+	if count = atomic.SwapInt64(p.hitsStats[metrics.CacheTag], 0); count > 0 {
+		if err = p.client.Count(metrics.MetricProcessResolverCacheHits, count, []string{metrics.CacheTag}, 1.0); err != nil {
+			return errors.Wrap(err, "failed to send process_resolver cache hits metric")
+		}
+	}
+
+	if count = atomic.SwapInt64(p.hitsStats[metrics.KernelMapsTag], 0); count > 0 {
+		if err = p.client.Count(metrics.MetricProcessResolverCacheHits, count, []string{metrics.KernelMapsTag}, 1.0); err != nil {
+			return errors.Wrap(err, "failed to send process_resolver kernel maps hits metric")
+		}
+	}
+
+	if count = atomic.SwapInt64(p.hitsStats[metrics.ProcFSTag], 0); count > 0 {
+		if err = p.client.Count(metrics.MetricProcessResolverCacheHits, atomic.SwapInt64(p.hitsStats[metrics.ProcFSTag], 0), []string{metrics.ProcFSTag}, 1.0); err != nil {
```

Comment:
```suggestion
		if err = p.client.Count(metrics.MetricProcessResolverCacheHits, count, []string{metrics.ProcFSTag}, 1.0); err != nil {
```
otherwise the SwapInt64 will return the `0` that was swapped in on line 270 -- or I've missed something :)

Reply:
Yup this was a typo 😄  thanks !